### PR TITLE
sass organizational changes

### PR DESCRIPTION
**Pull request by brianedelman**
Basic sass organizational changes and an overhaul of how we handle colors in sass and globally.

I am really torn on if I want to add [this ](https://github.com/Lightmatter/farewell_fax/blob/10f43522529b97bccb7dcfdec843dc57d782f282/tailwind.config.js#L70) in after seeing a recent PR on HTN. This approach works really well for colors IMO but the HTN PR has height and transition classes thrown in so it feels like over the life time of a project this file would grow and grow and grow.

I do think the current approach of commenting in the names is ugly but seeing how many potential classes the safelist could grow into, I am not sure this is better long term. Would love some thoughts around this before merging.

### CODE REVIEW

File: {{cookiecutter.repo_name}}/frontend/css/base/_forms.scss [hunk 1]
```diff
@@ -0,0 +1,291 @@
+/* stylelint-disable no-descending-specificity */
```
Comment (brianedelman) [inline {{cookiecutter.repo_name}}/frontend/css/base/_forms.scss:1]: NOTE: this file is just a copy paste from the current atoms

**Comment by brianedelman** [comment]
I am getting stylelint errors on sample app spinup but they are super confusing. Is this a known issue with template?
```

frontend/css/base/_forms.scss
 1:1  ✖  Unknown rule import-notation                import-notation
 1:1  ✖  Unknown rule keyframe-selector-notation     keyframe-selector-notation
 1:1  ✖  Unknown rule annotation-no-unknown. Did     annotation-no-unknown
         you mean function-no-unknown?

frontend/css/app.scss
 1:1  ✖  Unknown rule import-notation                import-notation
 1:1  ✖  Unknown rule keyframe-selector-notation     keyframe-selector-notation
 1:1  ✖  Unknown rule annotation-no-unknown. Did     annotation-no-unknown
         you mean function-no-unknown?

frontend/css/base/base.scss
 1:1  ✖  Unknown rule import-notation                import-notation
 1:1  ✖  Unknown rule keyframe-selector-notation     keyframe-selector-notation
 1:1  ✖  Unknown rule annotation-no-unknown. Did     annotation-no-unknown
         you mean function-no-unknown?

frontend/css/base.scss
 1:1  ✖  Unknown rule import-notation                import-notation
 1:1  ✖  Unknown rule keyframe-selector-notation     keyframe-selector-notation
 1:1  ✖  Unknown rule annotation-no-unknown. Did     annotation-no-unknown
         you mean function-no-unknown?

TypeError: result.replace is not a function
    at formatMessageText (/Users/brianedelman/.cache/pre-commit/repoteijj3vt/node_env-default/lib/node_modules/stylelint/lib/formatters/stringFormatter.js:187:5)
    at /Users/brianedelman/.cache/pre-commit/repoteijj3vt/node_env-default/lib/node_modules/stylelint/lib/formatters/stringFormatter.js:208:4
    at Array.map (<anonymous>)
    at formatter (/Users/brianedelman/.cache/pre-commit/repoteijj3vt/node_env-default/lib/node_modules/stylelint/lib/formatters/stringFormatter.js:199:42)
    at /Users/brianedelman/.cache/pre-commit/repoteijj3vt/node_env-default/lib/node_modules/stylelint/lib/formatters/stringFormatter.js:267:12
    at Array.reduce (<anonymous>)
    at module.exports (/Users/brianedelman/.cache/pre-commit/repoteijj3vt/node_env-default/lib/node_modules/stylelint/lib/formatters/stringFormatter.js:254:19)
    at Object.apply (/Users/brianedelman/.cache/pre-commit/repoteijj3vt/node_env-default/lib/node_modules/stylelint/node_modules/import-lazy/index.js:16:20)
    at prepareReturnValue (/Users/brianedelman/.cache/pre-commit/repoteijj3vt/node_env-default/lib/node_modules/stylelint/lib/prepareReturnValue.js:51:23)
    at standalone (/Users/brianedelman/.cache/pre-commit/repoteijj3vt/node_env-default/lib/node_modules/stylelint/lib/standalone.js:281:17)

frontend/css/base/_fonts.scss
 1:1  ✖  Unknown rule import-notation                import-notation
 1:1  ✖  Unknown rule keyframe-selector-notation     keyframe-selector-notation
 1:1  ✖  Unknown rule annotation-no-unknown. Did     annotation-no-unknown
         you mean function-no-unknown?

frontend/css/tailwind.css
 1:1  ✖  Unknown rule import-notation                import-notation
 1:1  ✖  Unknown rule keyframe-selector-notation     keyframe-selector-notation
 1:1  ✖  Unknown rule annotation-no-unknown. Did     annotation-no-unknown
         you mean function-no-unknown?

frontend/css/components/_links.scss
 1:1  ✖  Unknown rule import-notation                import-notation
 1:1  ✖  Unknown rule keyframe-selector-notation     keyframe-selector-notation
 1:1  ✖  Unknown rule annotation-no-unknown. Did     annotation-no-unknown
         you mean function-no-unknown?

frontend/css/components/_toasts.scss
 1:1  ✖  Unknown rule import-notation                import-notation
 1:1  ✖  Unknown rule keyframe-selector-notation     keyframe-selector-notation
 1:1  ✖  Unknown rule annotation-no-unknown. Did     annotation-no-unknown
         you mean function-no-unknown?
```

**Comment by gone** [comment]
no, it's not - wonder if it's a version not being locked down? I'll exlplore

**Comment by jake-kent** [comment]
replaced by #324 to make use of other file reorganization done in parallel

**Pull request closed without merging**